cs.json

```diff
@@ -0,0 +1,348 @@
+{
+    "name" : "Čeština",
+    "name-en": "Czech",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : false,
+            "dateFormat" : "[date]. [month] [year]",
+            "months" : [
+                "Leden",
+                "Únor",
+                "Březen",
+                "Duben",
+                "Květen",
+                "Červen",
+                "Červenec",
+                "Srpen",
+                "Září",
+                "Říjen",
+                "Listopad",
+                "Prosinec"
+            ],
+            "types" : {
+                "backpack" : "Batoh",
+                "emote" : "Akce",
+                "glider" : "Padák",
+                "emoji" : "Emotikon",
+                "loading" : "Načítací obrazovka",
+                "outfit" : "Oděv",
+                "pickaxe" : "Krumpáč",
+                "skydive" : "Stopa pádu",
+                "umbrella" : "Deštník",
+                "spray" : "Sprej",
+                "toy" : "Hračka",
+                "pet" : "Mazlíček",
+                "music" : "Hudba",
+                "wrap" : "Plášť",
+                "banner" : "Banner", //Could be banner or plakát, depending on where it is used
+                "bundle" : "Balíček",
+                "misc" : "Ostatní"
+            },
+            "rarity" : {
+                "frozen" : "Zamrzlý",
+                "lava" : "Lávový",
+                "legendary" : "Legendární",
+                "dark" : "Temný",
+                "marvel" : "Marvel",
+                "dc" : "DC",
+                "star_wars" : "Star Wars",
+                "gaming_legends" : "Herní Legendy",
+                "icon_series" : "Série Ikon",
+                "slurp" : "Slurp", //This is an interjection and it's hard to translate. Could be either Slint (Slintat in infinitive) as in drooling while drinking/eating or Srk (Srkat in infinitive) as in drinking a liquid loudly
```

Comment:
I took a look at the official German translation, which is "Schlürf" (from "schlürfen" in infinitive form), which means drinking a liquid loudly. So "Srk" probably fits better than "Slint". However, if you feel that users wouldn't understand that "Srk" is a translation for the Slurp item in Fortnite, I'd be fine with leaving it untranslated as "Slurp". Our goal with translations is to make the site more user-friendly, and nothing is gained if someone doesn't understand what a particular translation is about 🙂

Reply:
Let's keep it as Slurp then.